### ISSUE TITLE
HP Clinical Shipping not creating a box

### ISSUE THREAD

**Issue by wyattks** [episphere/connect#754]
During our test run with Erin at HP (in the Dev Dashboard), the shipping dashboard would not create a new box when trying to add the clinical collections to a box.  There were no boxes listed in the dropdown selection under "View Shipping Box Contents".  When she scanned in an ID she received a "Last created box is empty. Please add a specimen(s) to last box." message.  I had her look at inspect/console and did a screen capture.
[HP Clinical Test Shipping Issue screen capture.docx](https://github.com/episphere/connect/files/12820685/HP.Clinical.Test.Shipping.Issue.screen.capture.docx)

**Comment by JoeArmani** [comment]
**Update:**
This is likely the result of a batch update function I ran in dev testing. Continue monitoring. I do not expect it to present issues in normal shipping dashboard operations.

**Details:**
I can manually recreate this scenario in dev. It results from manual updates to Firestore (not shipping dashboard functionality), so I expect we'll be in a good spot moving forward.

Earlier in the dev cycle, I bulk-tested results when boxes were marked shipped in dev.

I ran a function to bulk set and retest these shipped and received settings on low-numbered boxes. The function was not site-specific, so all low-numbered boxes in dev were updated.

Shipped boxes are filtered in the shipping dashboard, so Box3 was found. It was empty but inaccessible when Erin tried to access it because of these manually updated settings.

Since this scenario was reproducible with manual updating but is not in line with how the dashboard functions, we should not see this issue in normal operation.
